Test/SelfHosting/Main.cs

```diff
@@ -0,0 +1,58 @@
+using Antlr4.Runtime;
```

Comment:
I'm confused. What is the purpose of the SelfHosting folder?

Reply:
This shouldn't have been checked in, sorry